Engine/src/Engine/Physics/OctTree.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <list>
+#include "glm/glm.hpp"
+#include "AABB.h"
+#include "Ray.h"
+#include "OctNode.h"
+
+namespace Engine
+{
+	/// <summary>
+	/// Forward declaration of Entity
+	/// </summary>
+	class Entity;
+
+	/// <summary>
+	/// 3D space is recursively divided into 8 portions

```

Comment:
I think a little more clarification here would be good. The world is divided into 8 portions that each handle a set of physics checks? Or is this to handle something else?

Reply:
Tried to make it more clear what an oct-tree is. If its still unclear, might just remove the explanation